tests/integration/individual_response/test_individual_response.py

```diff
@@ -1,14 +1,21 @@
-import re
+from datetime import datetime
 from unittest.mock import MagicMock
 
+from dateutil.tz import tzutc
+from freezegun import freeze_time
+
 from app import settings
 from app.publisher.exceptions import PublicationFailed
 from tests.integration.integration_test_case import IntegrationTestCase
 
 
+@freeze_time("2020-11-25T11:59:00")
 class IndividualResponseTestCase(IntegrationTestCase):
     def setUp(self):
         settings.EQ_INDIVIDUAL_RESPONSE_LIMIT = 2
+        settings.EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE = datetime.fromisoformat(
```

Comment:
I would have liked to have used the actual date this would happen for census (2021-04-28T14:00:00) however I ran into issues using freeze with session expirations on authentication (401). So the deadline was my lunch yesterday